index.d.ts

```diff
@@ -142,6 +157,11 @@ export function parse(query: string, options?: ParseOptions): ParsedQuery;
 export interface ParsedUrl {
 	readonly url: string;
 	readonly query: ParsedQuery;
+
+	/**
+	The fragment identifier of the URL.
```

Comment:
This needs to say when it's available. Same in the readme.

Reply:
This has been done individually for `parseURL` and `stringifyURL` sections

Reply:
It should be documented there too.

Reply:
It's not clear what you mean

Reply:
It's good now.